cmd/auth-provider-gcp/credentialconfig/README.md

```diff
@@ -0,0 +1 @@
+Temporarily copied from a PR intended to be merged into into kubernetes/kubernetes (https://github.com/kubernetes/kubernetes/pull/95775) - please do not modify this code!
```

Comment:
Not sure where your trying to say here.

Reply:
This is code from https://github.com/kubernetes/kubernetes/pull/95775 - eventually it will land, but in order to avoid a dependency on private code I put it here temporarily. I think it should probably left alone as much as possible so as to stay in sync with the PR. Maybe there might be a better way to indicate that info?

Reply:
Creating a temporary local copy of kubernetes/kubernetes#95775, so that we do not need to depend on K/K private libraries until kubernetes/kubernetes#95775 merges. Currently believe kubernetes/kubernetes#95775 will merge in v1.21. Please do not modify this code without checking with SIG cloud provider first.

Reply:
Marking as resolved since the README now mentions the PR should be merged in v1.21 in addition to mentioning the other information.